src/components/PeoplePage.tsx

```diff
@@ -15,17 +112,28 @@ export const PeoplePage = () => {
 
           <div className="column">
             <div className="box table-container">
-              <Loader />
-
-              <p data-cy="peopleLoadingError">Something went wrong</p>
-
-              <p data-cy="noPeopleMessage">
-                There are no people on the server
-              </p>
-
-              <p>There are no people matching the current search criteria</p>
-
-              <PeopleTable />
+              {loader ? (
+                <Loader />
+              ) : (
+                <>
+                  {persons.length === 0 ? (
+                    <p data-cy="noPeopleMessage">
+                      There are no people on the server
+                    </p>
+                  ) : (
+                    <>
+                      {filteredPersons.length === 0 ? (
+                        <p>There are no people matching the current search criteria</p>
+                      ) : (
+                        <PeopleTable
+                          personSlug={personSlug}
+                          persons={filteredPersons}
+                        />
```

Comment:
Consider moving this message into a PeopleTable because it`s a part of people's table flow.